packages/tracing/src/index.ts

```diff
@@ -1,10 +1,10 @@
-import { BrowserTracing } from './browser';
 import { addExtensionMethods } from './hubextensions';
-import * as TracingIntegrations from './integrations';
-
-const Integrations = { ...TracingIntegrations, BrowserTracing };
+import * as Integrations from './integrationsbundle';
 
 export { Integrations };
+
+export { BrowserTracing } from './browser';
```

Comment:
Ah hold on I know this is wrong, I’ll fix it in the morning.

Reply:
nvm this is why you don't read code right before bed - nothing is wrong here lol